.github/workflows/depthai.yml

```diff
@@ -40,8 +40,28 @@ jobs:
     runs-on: macos-10.15
     steps:
       - uses: bytedeco/javacpp-presets/.github/actions/deploy-macosx@actions
+  android-arm:
+    runs-on: ubuntu-18.04
+    container: centos:7
+    steps:
+      - uses: bytedeco/javacpp-presets/.github/actions/deploy-centos@actions
+  android-arm64:
+    runs-on: ubuntu-18.04
+    container: centos:7
+    steps:
+      - uses: bytedeco/javacpp-presets/.github/actions/deploy-centos@actions
+  android-x86:
+    runs-on: ubuntu-18.04
+    container: centos:7
+    steps:
+      - uses: bytedeco/javacpp-presets/.github/actions/deploy-centos@actions
+  android-x86_64:
+    runs-on: ubuntu-18.04
+    container: centos:7
+    steps:
+      - uses: bytedeco/javacpp-presets/.github/actions/deploy-centos@actions
```

Comment:
Please put everything for Android *before* the other platforms, in this file, cppbuild.sh, and pom.xml files as well.